.gitignore

```diff
@@ -33,4 +33,5 @@ coverage.txt
 # Emacs save files have literal pound signs/octothorpes/hashtags in them
 # So the \# escapes # as comment syntax
 \#*#
-.\#*
\ No newline at end of file
+.\#*
+sagalog/*
```

Comment:
On file locations, we are starting to generate some litter in various places. Not sure if there is an idiomatic way of doing this but I would prefer we either pick a dir path to place everything like ~/[scootdata], or pick a convention to have all dot files in ~/, like cloudscootaddr